src/main/java/techcourse/myblog/web/ArticleController.java

```diff
@@ -2,10 +2,82 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import techcourse.myblog.domain.Article;
 import techcourse.myblog.domain.ArticleRepository;
 
+import java.util.List;
+
 @Controller
 public class ArticleController {
     @Autowired
     private ArticleRepository articleRepository;
-}
+
+    @GetMapping("/")
+    private String getIndex(Model model) {
```

Comment:
`@GetMapping`을 통해 이미 메서드의 의미를 전달하고 있어요.

```suggestion
    private String index(Model model) {
```

Reply:
https://www.slipp.net/questions/79
위 글 참고해서 다른 메소드 명들도 수정했습니다.